import pytest


def test_fail():
    pytest.fail('test')
